README.md

```diff
@@ -40,7 +40,8 @@ harvest:
   email:     'email@example.com'
   password:  'password'
   subdomain: 'company'
```

Comment:
**question**: should we replace these three config values (email, password, and subdomain) with `token` in the README example?